test/Tick.t.sol

```diff
@@ -116,21 +116,21 @@ contract TickTest is Test, GasSnapshot {
     function testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForLowFee() public pure {
```

Comment:
i think we should add a test for when the min tick % tick spacing is 0 cause the number then shouldn't change from impl to impl

Reply:
there are 2 such tests already 
testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForMinTickSpacing
testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForMaxTickSpacing